### DISCUSSION TITLE
Is there any way to create multiple windows?

### DISCUSSION THREAD

**Discussion by atahabaki** [emilk/egui#983] [Q&A]
Hi, I need 2 or more windows, not `egui::containers::Window`, not inside the main window i mean. Seperate 2 windows?

Thanks!

**Comment by emilk** [comment]
`eframe` currently only supports a single native window.

**Comment by parasyte** [comment]
See also #1044

**Reply by atahabaki** [reply]
It's nice to see some improvements about that, really cool. Thanks for the reply.
